doc/api/next_api_changes/behavior/17494-JMK.rst

```diff
@@ -0,0 +1,24 @@
+Constrained layout rewrite
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The layout manager ``constrained_layout`` was re-written with different
+outer constraints that should be more robust to complicated subplot layouts.
+User-facing changes are:
+
+- some poorly constrained layouts will have different width/height plots than
+before.
+- colorbars now respect the ``anchor`` kwarg of `colorbar.make_axes`
+- colorbars are wider.
+- colorbars in different rows or columns line up more robustly.
+- ``hspace`` and ``wspace`` options to  `figure.set_constrained_layout_pads`
+were twice as wide as the docs said they should be.  So these now follow
+the docs.
+
+This feature will remain ``experimental'' until the new changes have been
```

Comment:
Smart quotes are off in our docs config.
```suggestion
This feature will remain "experimental" until the new changes have been
```

Reply:
oops, been writing in Latex too much lately :wink: